programs/mpl-staking/src/instructions/penalties/allow_claiming.rs

```diff
@@ -0,0 +1,32 @@
+use super::Penalty;
+use crate::cpi_instructions;
+use anchor_lang::prelude::*;
+use mplx_staking_states::error::MplStakingError;
+
+/// Restricts claiming rewards from the specified mining account.
```

Comment:
```suggestion
/// Allows claiming rewards from the specified mining account.
```

Reply:
My idea was to restrict claiming and withdraws at the same time because you can't withdraw if you have some unclaimed rewards. Albeit, a user might have no rewards and still violate the rules. Presumably, restricting withdrawal (while still restricting claiming) is a more sensible approach. wdyt?

Reply:
Yep, the logic is good. The comments are a bit off and need fixing

Reply:
Wdym? I've committed your suggestion, haven't I?